lib/features/composer/presentation/model/rich_text_option.dart

```diff
@@ -0,0 +1,30 @@
+
+import 'package:html_editor_enhanced/html_editor.dart';
+import 'package:model/mailbox/select_mode.dart';
+
+class RichTextOption {
+  final ButtonType buttonType;
+  final SelectMode selectMode;
```

Comment:
what is `selectMode` in this case? you mean `selected` item in list?

Reply:
yes